main.js

```diff
@@ -66,17 +80,21 @@ function decode({hashlink}) {
 /**
  * Verifies a hashlink resulting in a simple true or false value.
  *
- * @param {Object} options - The options for the encode operation.
- * @param {string} options.hashlink - The encoded hashlink value to verify.
- * @param {Uint8Array} options.data - Optional data to use when verifying
+ * @param {string} hashlink - The encoded hashlink value to verify.
+ * @param {Uint8Array} data - Optional data to use when verifying
  *   hashlink.
- * @param {Array} options.resolvers - An array of Objects with key-value
+ * @param {Array<object>} [resolvers] - An array of Objects with key-value
  *   pairs. Each object must contain a `scheme` key associated with a
  *   Function({url, options}) that resolves any URL with the given scheme
  *   and options to data.
  *
  * @returns {Promise<boolean>} true if the hashlink is valid, false otherwise.
  */
 async function verify({hashlink, data, resolvers}) {
+  if(!data) {
+    // TODO: Add fetching of data from url
+    throw new Error('Fetching of data from hashlink coming soon.');
```

Comment:
Same comment as above:
```suggestion
    throw new Error('Fetching of data from hashlink not implemented.');
```